mono/mini/mini-exceptions.c

```diff
@@ -2396,12 +2396,15 @@ mono_setup_altstack (MonoJitTlsData *tls)
 
 	g_assert ((guint8*)&sa >= (guint8*)tls->stack_ovf_guard_base + tls->stack_ovf_guard_size);
 
+/* the valloc call here seems problematic on AIX */
+#if !defined(_AIX)
```

Comment:
ifdefing this part out, makes this code broken and useless. instead we should rather not set `MONO_ARCH_SIGSEGV_ON_ALTSTACK` in `mini-ppc.h`.